.pre-commit-config.yaml

```diff
@@ -75,7 +75,7 @@ repos:
     hooks:
       - id: flake8
         exclude: "^app/core/migrations/|^app/data/migrations/|^app/core/scripts"
-        args: [--max-line-length=119, --max-complexity=4, --pytest-fixture-no-parentheses]
+        args: [--max-line-length=119, --max-complexity=4, --pytest-fixture-no-parentheses, --extend-ignore=EXE002]
```

Comment:
Note: This change is fine since flake8's file check of "executable scripts should start with shebang" is broken for Windows filesystem.

@drakeredwind01 has the repo checked out to his Windows filesystem D:\ drive and NTFS has no notion of the executable bit. So it tells flake8 that all files are executable, including ones like `app/core/admin.py` that's not executable under POSIX. So flake8 complains and says it should have a shebang, which is wrong.

Anyway, `pre-commit` has its own check for this that [does work with NTFS](https://github.com/pre-commit/pre-commit-hooks/pull/480) and we already have it enabled `- id: check-executables-have-shebangs`, so the flake8 one is redundant.